flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingExecutionVertex.java

```diff
@@ -37,4 +39,11 @@
      * @return state of the execution vertex
      */
     ExecutionState getState();
+
+    /**
+     * Get the {@link ConsumedPartitionGroup}s.
+     *
+     * @return list of {@link ConsumedPartitionGroup}s
+     */
+    List<ConsumedPartitionGroup> getConsumerPartitionGroups();
```

Comment:
`getConsumerPartitionGroups` -> `getConsumedPartitionGroups `

Reply:
Sorry for being careless. Resolved.